docs/Extensibility_UG/frontend_extensions.md

```diff
@@ -108,32 +113,57 @@ as it was demonstrated with ``TemplateExtension::Identity``.
 Attribute Mapping
 ++++++++++++++++++
 
-As described above, ``OpExtension`` is useful when attributes can be mapped one by one or initialized by a constant. 
+As described above, ``OpExtension`` is useful when attributes can be mapped one by one or initialized by a constant.
+Attributes in OpenVINO operators are identified by their names, so for frameworks that also have named attributes (like TensorFlow, PaddlePaddle, ONNX), 
+you can specify name to name mapping. For frameworks where OpenVINO operator's attributes can be mapped to one of the framework 
+operator inputs (like PyTorch), there's a name to input index mapping.
+
+Named attributes mapping
+^^^^^^^^^^^^^^^^^^^^^^^^
+
 If the set of attributes in framework representation and OpenVINO representation completely match by their names and types, 
-nothing should be specified in OpExtension constructor parameters. The attributes are discovered and mapped 
-automatically based on ``visit_attributes`` method that should be defined for any OpenVINO operation.
+no attribute mapping has to be specified in OpExtension constructor parameters. The attributes are discovered and mapped automatically 
+based on ``visit_attributes`` method that should be defined for any OpenVINO operation.
 
 Imagine you have CustomOperation class implementation that has two attributes with names: ``attr1`` and ``attr2``.
 
 .. doxygensnippet:: docs/snippets/ov_extensions.cpp
    :language: cpp
    :fragment: [frontend_extension_CustomOperation]
 
-And the original model in the framework representation also has operation named “CustomOperation” with the same 
+And original model in framework representation also has operation with name ``CustomOperation`` with the same 
 ``attr1`` and ``attr2`` attributes. Then with the following code:
 
 .. doxygensnippet:: docs/snippets/ov_extensions.cpp
    :language: cpp
    :fragment: [frontend_extension_CustomOperation_as_is]
 
-Both ``attr1`` and ``attr2`` are copied from framework representation to OpenVINO representation automatically. 
+Both ``attr1`` and ``attr2`` are copied from framework representation to OpenVINO representation automatically.
+
+The code is slightly different, if you want to use ``OpExtension`` for PaddlePaddle model. PaddlePaddle operations 
+have named inputs and outputs, so you need to pass their names in ``OpExtension`` constructor. For example, 
```

Comment:
First I think this description is not relevant to this particular paragraph that is dedicated to named attributes mapping, not inputs and outputs. So we need a chapter break or something similar here.

Second, when you are describing the demand to have named input/output mapping please add: `Named inputs (or outputs) are not ordered, so they don't have indices to be matched with the ordered inputs (or outputs) of OpenVINO operation.`. This emphases the real reason to have name to index mapping -- not just because they are named in Paddle, but because of absence of any order for such inputs and outputs.

Reply:
done